src/styles/primeng-theme/components/_datatable.scss

```diff
@@ -0,0 +1,54 @@
+// Variables
+$datatable-gray: $gray-lighter;
```

Comment:
All variables should go in this file:

https://github.com/dotCMS/core-web/pull/350/files#diff-020fe844182b21b4f7ee91ed255735ee